database/resources/events.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Hackathon",
+    "description": "What do I need to know for a hackathon?",
+    "url": "https://tips.hackathon.com/article/what-is-a-hackathon",
+    "category": "resources",
+    "subcategory": "events"
+  },
+  {
+    "name": "Your First Hackathon: A Survival Guide",
+    "description": "How To Prepare For Your First Hackathon.",
```

[Nitpick] This seems to repeat the source's title. Consider using the advice I mentioned [here](https://github.com/rupali-codes/LinksHub/pull/717#discussion_r1205580421) to help you revise it. 
```suggestion
    "description": "How To Prepare For Your First Hackathon.",
```